diagnostic.md

```diff
@@ -100,11 +125,26 @@ object.
 ```
 
 ```sh
-# your answer here
+curl --include --request PATCH \
+http://www.example/donuts/$ID \
+--header "Content-Type: application/json" \
+--data '{
+  "donut": {
+    "name": "Krüller"
+  }
+}'
 ```
 
 Write an AJAX request to change the donut on the server using JSON.
 
 ```js
-let changeDonut = /* your answer here */;
+let changeDonut = function(id, data){
```

Comment:
This will change an arbitrary existing donut object, but we are looking to change a specific donut with specific data.

```javascript
let changeDonut = function (donutID) {
  return $.ajax({
    url: 'http://example.com/donuts/' + donutID,
    method: 'PATCH',
    data: {
      "donut" {
        "name": "Krüller",
      }
    }
  });
};
```